website/docs/r/cis_waf_package.html.markdown

```diff
@@ -0,0 +1,61 @@
+---
+layout: "ibm"
+page_title: "IBM: ibm_cis_waf_package"
+sidebar_current: "docs-ibm-resource-cis-waf-package"
+description: |-
+  Provides a IBM CIS WAF Package resource.
+---
+
+# ibm_cis_waf_package
+
+Provides a IBM CIS WAF Package resource. This resource is associated with an IBM Cloud Internet Services instance and a CIS Domain resource. It allows to change WAF Package settings of a domain of a CIS instance
+
```

Comment:
UI names it as OWASP rule set.. Can this be termed as alternate name in the docs..